### ISSUE TITLE
Link text is missing for the Roleypoly logo link

### ISSUE THREAD

**Issue by emma-sg** [emma-sg/roleypoly#2]
When using a screen reader or other assistive device, the Roleypoly svg logo has no text associated with it. It is read as a blank link. The solution to this would simply be to add a `title` attribute to the embedded svg.
